buildpack.toml

```diff
@@ -3,7 +3,7 @@ api = "0.2"
 [buildpack]
 id = "salesforce/nodejs-fn"
 name = "Salesforce NodeJS Function Structured Middleware Buildpack"
-version = "1.6.1"
+version = "1.6.2"
```

Comment:
Is a minor update applicable here? I know we haven't been too strict, but a potential `@salesforce/core` upgrade to `2.10.0` for _every_ function feels like it might be risker.

Reply:
I could be convinced either way.

Reply:
I was torn, but I could go either way as well.